examples/gps_simpletest.py

```diff
@@ -84,13 +85,13 @@
         print("Latitude: {0:.6f} degrees".format(gps.latitude))
         print("Longitude: {0:.6f} degrees".format(gps.longitude))
         print(
-            "Precise Latitude: {:2.}{:2.4f} degrees".format(
-                gps.latitude_degrees, gps.latitude_minutes
+            "Precise Latitude: {:3.0f} degs, {:2.4f} mins".format(
+                math.floor(gps.latitude_degrees), gps.latitude_minutes
             )
         )
         print(
-            "Precise Longitude: {:2.}{:2.4f} degrees".format(
-                gps.longitude_degrees, gps.longitude_minutes
+            "Precise Longitude: {:3.0f} degs, {:2.4f} mins".format(
+                math.floor(gps.longitude_degrees), gps.longitude_minutes
```

Comment:
I don't think you need the float specifier for the degrees if you're using `math.floor()` right?